File: Java/DayTwo/TaskOne.java
public class TaskOne{

public static int checkfatherToSonAge(int fatherAge, int sonAge) {
int fatherPastAge = fatherAge - sonAge * 2;

	if (fatherPastAge < 0) {
		return 0;
	}

 return  fatherPastAge;
}

public static void main(String... makaveli) {
System.out.print(checkfatherToSonAge(50,16));

}
}

File: JavaScript/DayTwo/TaskOne.js
function checkFatherToSonAge(fatherAge, sonAge) {
	let fatherFutureAge = fatherAge -  sonAge * 2;
		if (fatherFutureAge < 0) {
			return 0;
		}
return fatherFutureAge;

}

console.log(checkFatherToSonAge(50,16));